include/sdf/Mesh.hh

```diff
@@ -104,6 +108,12 @@ namespace sdf
     /// \param[in] _center True to center the submesh.
     public: void SetCenterSubmesh(const bool _center);
 
+    /// \brief Calculate and return the Mass Matrix values for the Mesh
+    /// \param[in] density Density of the mesh in kg/m^3
+    /// \return A std::optional with gz::math::Inertiald object or std::nullopt
```

Comment:
```suggestion
    /// \param[in] _density Density of the mesh in kg/m^3
    /// \param[in] _calculatorParams
    /// \param[in] _config
    /// \return A std::optional with gz::math::Inertiald object or std::nullopt
```

Reply:
Added in 5e3d708e32db1531b64f950317f40345735196e2